vhdbuilder/packer/install-dependencies.sh

```diff
@@ -543,26 +543,48 @@ while IFS= read -r imageToBePulled; do
 done <<< "$ContainerImages"
 echo "Waiting for container image pulls to finish. PID: ${image_pids[@]}"
 wait ${image_pids[@]}
+capture_benchmark "${SCRIPT_NAME}_caching_container_images"
+
+retagAKSNodeCAWatcher() {
```

Comment:
I feel we should define these either in a separate file or either at the top/bottom of this file